modules/Data_Cleaning/lab/Data_Cleaning_Lab_Key.Rmd

```diff
@@ -148,13 +148,15 @@ BloodType %>% count(type)
 
 6. Make a new tibble of `BloodType` called `Bloodtype_split` that splits the `type` variable into two called `blood_type` and  `Rhfactor`. Note: periods are special characters that generally are interpreted as wild cards thus we need `\\` to tell R that we want it to be interpreted as a period.
```

Comment:
6. Make a new tibble of `BloodType` called `Bloodtype_split` that splits the `type` variable into two called `blood_type` and  `Rhfactor`. Note: periods are special characters that generally are interpreted as wild cards thus we need `\\.` instead of simply "." to tell R that we want it to be interpreted as a period.

Reply:
Does that work, @avahoffman ?

Reply:
yep!

Reply:
cool :)